editor/editor_node.cpp

```diff
@@ -238,185 +238,182 @@ void EditorNode::_unhandled_input(const Ref<InputEvent> &p_event) {
+				opening_prev = false;
 
-		if (unsaved_cache != (saved_version != editor_data.get_undo_redo().get_version())) {
+			if (unsaved_cache != (saved_version != editor_data.get_undo_redo().get_version())) {
 
-			unsaved_cache = (saved_version != editor_data.get_undo_redo().get_version());
-			_update_title();
-		}
+				unsaved_cache = (saved_version != editor_data.get_undo_redo().get_version());
+				_update_title();
+			}
 
-		if (last_checked_version != editor_data.get_undo_redo().get_version()) {
-			_update_scene_tabs();
-			last_checked_version = editor_data.get_undo_redo().get_version();
-		}
+			if (last_checked_version != editor_data.get_undo_redo().get_version()) {
+				_update_scene_tabs();
+				last_checked_version = editor_data.get_undo_redo().get_version();
+			}
 
-		// update the animation frame of the update spinner
-		uint64_t frame = Engine::get_singleton()->get_frames_drawn();
-		uint32_t tick = OS::get_singleton()->get_ticks_msec();
+			// update the animation frame of the update spinner
+			uint64_t frame = Engine::get_singleton()->get_frames_drawn();
+			uint32_t tick = OS::get_singleton()->get_ticks_msec();
 
-		if (frame != update_spinner_step_frame && (tick - update_spinner_step_msec) > (1000 / 8)) {
+			if (frame != update_spinner_step_frame && (tick - update_spinner_step_msec) > (1000 / 8)) {
 
-			update_spinner_step++;
-			if (update_spinner_step >= 8)
-				update_spinner_step = 0;
+				update_spinner_step++;
+				if (update_spinner_step >= 8)
+					update_spinner_step = 0;
 
-			update_spinner_step_msec = tick;
-			update_spinner_step_frame = frame + 1;
+				update_spinner_step_msec = tick;
+				update_spinner_step_frame = frame + 1;
 
-			// update the icon itself only when the spinner is visible
-			if (EditorSettings::get_singleton()->get("interface/editor/show_update_spinner")) {
-				update_spinner->set_icon(gui_base->get_icon("Progress" + itos(update_spinner_step + 1), "EditorIcons"));
+				// update the icon itself only when the spinner is visible
+				if (EditorSettings::get_singleton()->get("interface/editor/show_update_spinner")) {
+					update_spinner->set_icon(gui_base->get_icon("Progress" + itos(update_spinner_step + 1), "EditorIcons"));
+				}
 			}
-		}
-
-		editor_selection->update();
 
-		scene_root->set_size_override(true, Size2(ProjectSettings::get_singleton()->get("display/window/size/width"), ProjectSettings::get_singleton()->get("display/window/size/height")));
+			editor_selection->update();
 
-		ResourceImporterTexture::get_singleton()->update_imports();
-	}
-	if (p_what == NOTIFICATION_ENTER_TREE) {
+			scene_root->set_size_override(true, Size2(ProjectSettings::get_singleton()->get("display/window/size/width"), ProjectSettings::get_singleton()->get("display/window/size/height")));
 
-		Engine::get_singleton()->set_editor_hint(true);
+			ResourceImporterTexture::get_singleton()->update_imports();
+		} break;
+		case NOTIFICATION_ENTER_TREE: {
```

Comment:
Maybe add an empty line before this one, as done for other cases? That does make the code slightly more readable IMO with such big code blocks.